js/views/roomlistview.js

```diff
@@ -103,7 +103,8 @@
 		templateContext: function() {
 			return {
 				isDeletable: (this.model.get('participantType') === 1 || this.model.get('participantType') === 2) &&
-					(Object.keys(this.model.get('participants')).length > 1 || this.model.get('numGuests') > 0)
+					(Object.keys(this.model.get('participants')).length > 1 || this.model.get('numGuests') > 0),
+				numUnreadMessages: this.model.get('unreadMessages') > 99 ? '99+' : this.model.get('unreadMessages')
```

Comment:
I think that the `99+` should be internationalized, as a different string could be needed (although I am not really sure) in languages that use a non-latin alphabet.

Reply:
I quickly thought about it too, but I wouldn't know which language does not use + for +? :D 
lets keep it until someone complains.